APP/model.js

```diff
@@ -15,6 +15,29 @@ FROM
     });
 };
 
+exports.selectArticles = () => {
+  return db
+    .query(
+      `SELECT articles.author, 
+    articles.title, 
+    articles.article_id, 
+    articles.topic, 
+    articles.created_at, 
+    articles.votes, 
+    articles.article_img_url,
+    COUNT(comments.comment_id) AS comment_count
+FROM articles
+LEFT JOIN comments ON articles.article_id = comments.article_id
+GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url
+ORDER BY articles.created_at DESC
```

Comment:
Pristine SQL 🥇

Reply:
Might be worth using CAST to coerce the comment count into a number instead of a string (I think it's currently a string/I might be wrong)